Example/Authenticator/View Controllers/Connection/ConnectionsViewController.swift

```diff
@@ -225,6 +225,32 @@ extension ConnectionsViewController: ConnectionCellEventsDelegate {
     }
 
     func deletePressed(id: String, showConfirmation: Bool) {
+        checkInternetConnection(id: id, showConfirmation: showConfirmation)
+    }
+
+    func reconnectPreseed(id: String) {
+        viewModel.reconnect(id: id)
+    }
+
+    func consentsPressed(id: String) {
+        viewModel.consentsPressed(connectionId: id)
+    }
+    
+    private func checkInternetConnection(id: String, showConfirmation: Bool) {
```

Comment:
And now this method not only checks the internet connection. It also removes the connection. You better rename it to `checkInternetConnectionAndRemove`, cause now it confuses.